app/controllers/application_controller.rb

```diff
@@ -3,5 +3,13 @@ class ApplicationController < ActionController::Base
 
   before_action :authenticate_user!
 
+  def after_sign_in_path_for(current_user)
+    if current_user.profile.blank?
+      new_profile_path
+    else
+      current_user
```

Comment:
Set `root`in routes to `users_path` and redirect to `root_path`in the else clause